apps/nova/src/app/[locale]/(marketing)/logo-title.tsx

```
@@ -0,0 +1,21 @@
+'use client';
+
+import {cn} from '@/lib/utils';
+// import { useParams } from 'next/navigation';
+
+export default function LogoTitle({
+
+}: {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)